menu/menu.yaml

```
@@ -381,11 +381,11 @@
       Title: Microsoft Generic Host
     - Url: nservicebus/hosting/web-application
       Title: Web Application
-    - Url: nservicebus/hosting/azure-functions
-      Title: Azure Functions
+    - Url: nservicebus/hosting/azure-functions-service-bus
+      Title: Azure Functions ServiceBus
       Articles:
-      - Url: nservicebus/hosting/azure-functions/service-bus
-        Title: Service Bus
+      - Url: nservicebus/hosting/azure-functions-service-bus/custom-triggers
+        Title: Custom Azure Function triggers
```

could be shortened given this is already under Azure Functions?

```suggestion
        Title: Custom triggers
```